runtimerevolution/web3-roulette
# [WEB3-32] User data management

# Description

This PR updates the way the app manages user data. The google api access token is saved into local storage, a request is made on each page rendering to get the user information which is passed using context api.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-32

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Test login and logout
- Navigate through the different pages
- Verify different user roles

## Checklist:
